### PR TITLE
Adding warnings to create org modal (PROJQUAY-4511)

### PR DESCRIPTION
**Pull request by bcaton85**
Previous Quay UI had additional warnings when trying to create and organization. The conditions are:
Condition `invalid org name` displays error `Must be alphanumeric, all lowercase, at least 2 characters long and at most 255 characters long`.
Condition `name less than 4 or more than 30` displays warning `Namespaces less than 4 or more than 30 characters are only compatible with Docker 1.6+`.
Condition `name contains dashes or dots` displays warning `Namespaces with dashes or dots are only compatible with Docker 1.9+`.

### CODE REVIEW

File: src/routes/OrganizationsList/CreateOrganizationModal.tsx [hunk 1]
```diff
@@ -14,21 +14,55 @@ import FormError from 'src/components/errors/FormError';
...
+      setValidation({
+        message:
+          'Namespaces less than 4 or more than 30 characters are only compatible with Docker 1.6+',
+        isValid: true,
+        type: 'warning',
+      });
+    } else if (value.includes('.') || value.includes('-')) {
+      setValidation({
+        message:
+          'Namespaces with dashes or dots are only compatible with Docker 1.9+',
```
Comment (harishsurf) [inline src/routes/OrganizationsList/CreateOrganizationModal.tsx:59]: this warning did not show up for me. It's probably getting past the regex check

**Review by harishsurf — COMMENTED** [review]
looks good! left a comment

File: src/routes/OrganizationsList/CreateOrganizationModal.tsx (hunk 1 above)
Comment (bcaton85) [reply src/routes/OrganizationsList/CreateOrganizationModal.tsx:59]: do you have the name you used? I can test it out to see what the issue is.

Comment (harishsurf) [reply src/routes/OrganizationsList/CreateOrganizationModal.tsx:59]: huh, I seems to be working now. Initially I didn't see the warning for any name that had a `.` or `-`

**Review by harishsurf — APPROVED** [review]

**Pull request merged**